test/system/errorService/service.js

```diff
@@ -14,12 +14,11 @@
 
 'use strict'
 
-var mu = require('../../../core/core')()
-var mue = require('mu-error')()
+var mu = require('../../../packages/mu')()
```

Comment:
that's not how it works - packages/mu is the core mu module that will be published 

apparatus/mu is the monorepo - the package.json file there is purely for dev env (notice it's private)